pkg/port/iscsit/iscsid.go

```diff
@@ -948,4 +949,10 @@ func (s *ISCSITargetDriver) UpdateStats(conn *iscsiConnection) {
 		s.TargetStats.TotalWriteBlockCount += int64(conn.resp.ExpectedDataLen)
 		break
 	}
+	if _, ok := s.TargetStats.SCSIIOCount[(int)(conn.resp.SCSIOpCode)]; ok != false {
+		s.TargetStats.SCSIIOCount[(int)(conn.resp.SCSIOpCode)] += 1
+	} else {
+		s.TargetStats.SCSIIOCount[(int)(conn.resp.SCSIOpCode)] = 1
```

Comment:
remove entire if else block and just write `s.TargetStats.SCSIIOCount[(int)(conn.resp.SCSIOpCode)] += 1` if the key is not there it will be added, else incremented.

Reply:
https://play.golang.org/p/kJdBaYOvoLe